MetadataMigration/src/test/java/org/opensearch/migrations/EndToEndTest.java

```diff
@@ -37,15 +37,15 @@
 class EndToEndTest extends BaseMigrationTest {
 
     private static Stream<Arguments> scenarios() {
```

Comment:
This is pretty messy, can we refactor so `SupportedClusters` returns a definitive list of source and targets?  Then this function only adds test specific properties?